adoc/helm_chart_override.adoc

```diff
@@ -13,7 +13,8 @@ You can adjust the {helm} chart installation in one of the following ways:
 $ helm install <RELEASE_NAME> \
 oci://registry.suse.com/private-registry/private-registry-helm \
 --set harborAdminPassword=<MY_PASSWORD> \
---set externalURL=https://my-harbor.domain
+--set externalURL=https://my-harbor.domain \
+--set expose.ingress.hosts.core=<MY_PRIVATE_REGISTRY_FQDN>
```

Comment:
Are `<MY_PRIVATE_REGISTRY_FQDN>` and `my-harbor.domain` not supposed to be the same?

I mean, maybe this should say:

```suggestion
--set externalURL=https://<MY_PRIVATE_REGISTRY_FQDN> \
--set expose.ingress.hosts.core=<MY_PRIVATE_REGISTRY_FQDN>
```

But maybe I'm wrong.